docs/notes-pa.md

```diff
@@ -3,4 +3,166 @@ how the permutation argument is formed.
 
 
 Permutation argument
-==============
\ No newline at end of file
+==============
+
+Within PLONK, the permutation 
+polynomials are used to form 
+arugments about the proving 
+scheme. 
+
+These arguments are used to 
+check samples of elements, 
+against the original ones. 
+As is noted in other parts
+of the docs, there is the 
+need for a copy check in 
+in SNARK schemes. This is 
+crucial in our zk-SNARK
+scheme, to ensure the 
+correctness of wires and 
+allow us to express circuits
+with less differing vectors 
+per polynomial. 
+
+PLONK uses an argument to check 
+the correctness of a shuffle in 
+its proving algorithm. This allows 
+for the checking of individual 
+wires rather than entire sets. 
+In principle, PLONK can compare sets 
+of polynomials for equality by 
+comparing an `n` degree polynomial 
+with the same `n` degree polynomial 
+where some elements have been 
```

Comment:
Here it seems that you are conflating sets with polynomials. I think here you probably mean multi-sets of polynomials, because if the polynomials are the same, then they would just be duplicates. 

Can you elaborate on what you mean by "sets of polynomials" and elements have been rotated" ?